web/src/stream_settings_data.ts

```diff
@@ -37,6 +44,7 @@ export function get_sub_for_settings(sub: StreamSubscription): SettingsSubscript
             "dayofyear_year",
         ),
         creator: stream_data.maybe_get_creator_details(sub.creator_id),
+        is_active,
```

Comment:
here we can write same thing in less and straight forward code
```suggestion
        is_active: creator ? people.is_person_active(creator.user_id) : false,
```